checker/bin/wpi.sh

```diff
@@ -33,9 +33,9 @@ SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 SCRIPTPATH="${SCRIPTDIR}/wpi.sh"
 
 # Report line numbers when the script fails, from
-# https://unix.stackexchange.com/a/522815
+# https://unix.stackexchange.com/a/522815 .
```

Comment:
What is the purpose of this change?

Reply:
That period appears in wpi-many.sh but not in this file.  I made the files more consistent, so that their diffs are smaller and it's easier to compare them and to copy changes from one to the other.